os/arch/arm/src/armv7-a/cp15_cacheops.h

```diff
@@ -156,30 +156,30 @@
 
 /* Set/way format */
 
-#define CACHE_WAY_SHIFT (3)     /* Bits 30-31: Way in set being accessed */
+#define CACHE_WAY_SHIFT (3)		/* Bits 30-31: Way in set being accessed */
```

Comment:
check again. many mistakes till line  560

Reply:
I will check and update